addons/crm_claim/calendar_event.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+import datetime
+
+from openerp import api, fields, models
+
+class CalendarEvent(models.Model):
+    _inherit = "calendar.event"
+
+    claim_id = fields.Many2one('crm.claim', string="Claim", ondelete='cascade')
+    start_datetime = fields.Datetime(default= lambda self: fields.Datetime.now())
```

Comment:
yes you are right. lambda not required here.and directly (default= fields.Datetime.now)